html/components/team-time-tab.js

```diff
@@ -1145,10 +1145,11 @@ function createTimeoutDialog() {
 		}
 		if (v != null && row.length == 0) {
 			var jam = Number(WS.state[prefix+'.PrecedingJamNumber']);
-			var dur = isTrue(WS.state[prefix+'.Running']) ? 'Running' : _timeConversions.msToMinSec(WS.state[prefix+'.Duration']);
+			var dur = isTrue(WS.state[prefix+'.Running']) ? 'Running' :	_timeConversions.msToMinSec(WS.state[prefix+'.Duration'], true);
```

Comment:
Is it right to round these up? Timeouts (usually) use a count up clock, so this can display a different value to what was on the scoreboard.

Reply:
The original behaviour was to always round up when we are not dealing with a value straight from a clock. I decided to keep that for now.

Given that the behaviour for clocks is to round opposite to the running direction it would probably make sense to round any duration down, but I'd consider that a separate change from this bugfix.